[abseil-cpp] Add missing linkopts setting to a cctz library

This ensures that Drake's subsequent patch_cmds script to set all
libraries to static linking will indeed find all libraries.

Submitted upstream as https://github.com/google/cctz/pull/305 but
Google requested to implement this themselves a different way (with
Copybara magic).

If we don't see them fix it the next few months, maybe we can try re-
pinging them.

--- absl/time/internal/cctz/BUILD.bazel
+++ absl/time/internal/cctz/BUILD.bazel
@@ -34,6 +34,7 @@ cc_library(
     hdrs = [
         "include/cctz/civil_time.h",
     ],
+    linkopts = [],
     textual_hdrs = ["include/cctz/civil_time_detail.h"],
     visibility = ["//visibility:public"],
     deps = ["//absl/base:config"],
